### ISSUE TITLE
Cmd+k not working on mac in screen reader mode

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#19979]
Testing -177735

While testing the TPI I was unable to complete this step: `The hover is keyboard accessible via ctrl+k, ctrl+i (or cmd+k, cmd+i) when either the terminal or the terminal tab is focused`. The steps I tried were
1. Going into screen reader optimized mode on vscode
2. trying all 4 options ( ctrl+k, ctrl+i (or cmd+k, cmd+i) ) but cmd+k should be the right one
